app/services/concerns/xccdf/rule_references_rules.rb

```diff
@@ -54,17 +54,16 @@ def new_op_rule_references_rules
       end
 
       def rule_references_for(op_rule: nil)
-        label_hrefs = op_rule.rule_references.map do |rr|
-          [rr.label, rr.href]
-        end
+        @cached_references ||= @rule_references.index_by { |rr| [rr.label, rr.href] }
 
-        @rule_references.select do |reference|
-          label_hrefs.include?([reference.label, reference.href])
-        end
+        op_rule.rule_references.map do |rr|
```

Comment:
`op_rule` can be `nil` (as per fn signature).

Reply:
https://github.com/RedHatInsights/compliance-backend/pull/1140